packages/popover/src/main.vue

```diff
@@ -120,6 +120,10 @@ export default {
   },
 
   methods: {
+    doPrivateDestroy() {
+      this.doDestroy();
+      this.$emit('distroyed');
```

Comment:
`destroyed`. The documentation will also need to be updated.

Can you rely on the existing `hide` event instead (emitted on line 64 above)? Why is this needed?

Reply:
*Hide* triggered immediate, when transition don't finish yet! But sometimes need call action after full hide *popover*.

Reply:
>>The documentation will also need to be updated.

But where?

Reply:
@kolesoffac It's [here](https://github.com/ElemeFE/element/blob/dev/examples/docs/en-US/popover.md).